### ISSUE TITLE
Kodi/Local for shows : no difference

### ISSUE THREAD

**Issue by deepdish1** [jez500/chorus2#11]
Hi ,

If I go and select a show episode, whatever I choose local or kodi, the episode will be played on the kodi.
As far as it seems that local for video does not work great, maybe to remove it ?

**Comment by jez500** [comment]
Button is just broken, needs the correct action attached.

**Comment by un1versal** [comment]
this needs closing as it was fixed already

```
* Ability to toggle what addons are enabled
* Fixed back button in file browser #40
* Fixed remote toggle bug #39
* Settings improvements (addon selection)
* Added fanart background to popup video player
* Added bb code formatting to file/addon browser
* Playing a video respects kodi/local context #11
* Info toggles osd when playing #57
```
